provisioner/terraform/testdata/rich-parameters-validation/rich-parameters-validation.tf

```diff
@@ -0,0 +1,51 @@
+terraform {
+  required_providers {
+    coder = {
+      source  = "coder/coder"
+      version = "0.8.2"
+    }
+  }
+}
+
+data "coder_parameter" "number_example_min_max" {
+  name    = "number_example_min_max"
+  type    = "number"
+  default = 4
+  validation {
+    min = 3
+    max = 6
+  }
+}
+
+data "coder_parameter" "number_example_min" {
+  name    = "number_example_min"
+  type    = "number"
+  default = 4
+  validation {
+    min = 3
+  }
+}
+
+data "coder_parameter" "number_example_max" {
+  name    = "number_example_max"
+  type    = "number"
+  default = 4
+  validation {
+    max = 6
```

Comment:
I'd like to see test cases where the min is explicitly 0 and where the max is explicitly 0.

Reply:
Added